gapic/src/main/com/google/gapid/widgets/TabComposite.java

```diff
@@ -1091,18 +1376,51 @@ private Hover(Type type, Group group, int index, int cursor, Folder folder, Tab
       this.tab = tab;
     }
 
+    @Override
+    public boolean equals(Object o) {
+      if (this == o) {
+        return true;
+      }
+      if (o == null) {
+        return false;
+      }
+      if (getClass() != o.getClass()) {
+        return false;
+      }
```

Comment:
```suggestion
      if (this == o) {
        return true;
      } else if (!(o instanceof Hover)) {
        return false;
      }
```

That's the pattern we've been using everywhere else.

Reply:
Sure, done